### DISCUSSION TITLE
How do I update an outdated Kade Engine on FNF mods.

### DISCUSSION THREAD

**Discussion by fnflover7** [KadeArchive/Kade-Engine#2624] [Q&A] [answered]
So I was going to play some Friday Night Funkin' x Dusttrust. When i opened it I let it load after I let it load I saw my Kade Engine was outdated. Can anyone help me with this?    
|
|
|
v
Link
https://gamebanana.com/mods/313270](url)
^
 |
 |
 |

**Comment by GamingInfinite** [comment, marked as answer]
the easiest way rn is to use the newly implemented mod API.  It is surprisingly powerful for how early a stage it's in.  The only thing you won't get is the animations or stages.

**Reply by fnflover7** [reply]
damn

**Reply by fnflover7** [reply]
Need someone to do that for me I have no idea how to do it.

**Reply by GamingInfinite** [reply]
I've been trying my hand at it, trying to put together basically as many mods as I can in modern kade engine.  I'll put it next on the list how about that.  And then I'll start a repository and link it here.

**Reply by GamingInfinite** [reply]
https://gamejolt.com/games/Xdusttrust/643614 was this the mod in question?

**Reply by fnflover7** [reply]
Yes

**Reply by GamingInfinite** [reply]
Alright well I got all the files in the right places, but for some reason it's crashing when it gets to the freeplay menu.  Until my request for a log file so that mods can see whats going wrong where then I don't know how to fix it.  I'll release my progress in a bit and tinker for a while longer.  Also don't expect it to be a full port.  I don't know how to do modcharts, or have a way to implement custom notes.  Yet.  I will keep working on this along with the other ports I have in my list rn.

**Reply by fnflover7** [reply]
kk

**Reply by GamingInfinite** [reply]
I figured out why it wasn't working.  I forgot to add the meta files for the songs :/ .  Anyway I'll leave a link for my github repo once all the files are uploaded

**Reply by GamingInfinite** [reply]
https://github.com/GamingInfinite/KadeEngineMods
